services/src/unit-test/java/uk/gov/moj/sdt/services/UpdateRequestServiceTest.java

```diff
@@ -63,45 +63,67 @@
 import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 import uk.gov.moj.sdt.utils.SdtContext;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.*;

```

Comment:
Generally good practice to explicitly state the imports that are required rather than using a wildcard import. Replace this with the explicit imports that are needed.

Reply:
done

Reply:
Confirmed imports are now all explicit.